apps/platform/messages/en.json

```diff
@@ -71,18 +71,30 @@
   },
   "faq": {
     "title": "Frequently asked questions",
-    "voting": "Why are we voting with donations?",
-    "votingAnswer": "A new voting mechanism is used, called Quadratic Funding. The project with most donations will get the most funding from the City. It allows anyone to vote by donating money to their favourite projects. With every donation, funding is given to project from the matching pool.",
-    "city": "Why does the city use Quadratic funding instead of simple voting?",
-    "cityAnswer": "A new voting mechanism is used, called Quadratic Funding. The project with most donations will get the most funding from the City. It allows anyone to vote by donating money to their favourite projects. With every donation, funding is given to project from the matching pool.",
-    "vote": "How do I vote / donate?",
-    "voteAnswer": "A new voting mechanism is used, called Quadratic Funding. The project with most donations will get the most funding from the City. It allows anyone to vote by donating money to their favourite projects. With every donation, funding is given to project from the matching pool.",
-    "canVote": "Who can vote?",
-    "canVoteAnswer": "A new voting mechanism is used, called Quadratic Funding. The project with most donations will get the most funding from the City. It allows anyone to vote by donating money to their favourite projects. With every donation, funding is given to project from the matching pool.",
-    "decide": "Who decides which proposals will be accepted for voting and how?",
-    "decideAnswer": "A new voting mechanism is used, called Quadratic Funding. The project with most donations will get the most funding from the City. It allows anyone to vote by donating money to their favourite projects. With every donation, funding is given to project from the matching pool.",
-    "should": "Why should I donate?",
-    "shouldAnswer": "A new voting mechanism is used, called Quadratic Funding. The project with most donations will get the most funding from the City. It allows anyone to vote by donating money to their favourite projects. With every donation, funding is given to project from the matching pool."
+    "zazelenimo": "What is Zazelenimo?",
+    "zazelenimoAnswer1": "Let's go green is an online application for the implementation of participatory budgeting programs in cities.",
+    "zazelenimoAnswer2": "The special feature of Zazelenimo, compared to other applications that have the same purpose, is the use of Ethereum and a special form of voting called square funding.",
```

Comment:
'Zazelenimo' used here, but 'Let's go green' above? Is that ok?

Reply:
@tomislavmamic will you provide English translation and then we can be sure about naming :)

Reply:
Let's just use Zazelenimo everywhere. The rest of translation is good to go.